test/test_mdlp.py

```diff
@@ -195,5 +187,4 @@ def test_mdlp_iris():
         [ 2.,  1.,  2.,  2.],
         [ 2.,  0.,  2.,  2.]]
 
-    assert_array_equal(transformed, expected,
-                       err_msg="MDLP output is inconsistent with previous runs.")
+    assert (transformed == expected).all()
```

Comment:
`assert_array_equal`?

Reply:
ya sorry i dont know why i changed all the tests. i did this before getting things to run and should have been more defensive with changes. I will add the original test code back.

Reply:
ah actually i remember now that sklearn.utils.testing itself depends on nose. would you like to retain nose dependency (i will add it to requirements-test.txt) or to just remove the trivial wrappers that sklearn.utils.testing offers around checking equality appropritely?

Reply:
ah nm again. the sklearn functions here were just transitively imported from numpy, i imported them directly instead.